sequencer/sequencer.go

```diff
@@ -91,16 +90,26 @@ func (s *Sequencer) Start(ctx context.Context) {
 	tickerSendSequence := time.NewTicker(s.cfg.WaitPeriodSendSequence.Duration)
 	defer tickerProcessTxs.Stop()
 	defer tickerSendSequence.Stop()
+	// select transactions from the pool to create sequences
 	go func() {
 		for {
 			s.tryToProcessTx(ctx, tickerProcessTxs)
 		}
 	}()
+	// persist sequences into the DB
 	go func() {
 		for {
-			s.tryToSendSequence(ctx, tickerSendSequence)
+			s.tryToCreateSequence(ctx, tickerProcessTxs)
 		}
 	}()
+	// send and monitor persisted sequences to L1
+	go func() {
+		for {
+			s.txManager.SyncPendingSequences()
+			waitTick(ctx, tickerSendSequence)
+		}
+	}()
+
```

Comment:
I thought the ethtxmanager should be autonomous

Reply:
I've explained this here: https://github.com/0xPolygonHermez/zkevm-node/issues/1271#issuecomment-1288968113